lib/paper_trail/config.rb

```diff
@@ -40,7 +40,8 @@ def track_associations
 
     # Indicates whether PaperTrail is on or off.
     def enabled
-      PaperTrail.paper_trail_store[:paper_trail_enabled].nil? || PaperTrail.paper_trail_store[:paper_trail_enabled]
+      PaperTrail.paper_trail_store[:paper_trail_enabled].nil? ||
+        PaperTrail.paper_trail_store[:paper_trail_enabled]
```

Comment:
Nevermind, I'm looking at `RequestStore.fetch`, but your suggestion uses `Hash#fetch` after all. I'll try it.